core/server/common/src/main/webapp/browse-body.jsp

```diff
@@ -57,142 +39,20 @@
-                <tr>
-                  <th>
-                    <% if (fileInfo.getIsDirectory()) { %>
-                      <i class="icon-folder-close"></i>
-                    <% } %>
-                    <% if (!fileInfo.getIsDirectory()) { %>
-                      <i class="icon-file"></i>
-                    <% } %>
-                    <a href="<%= (request.getAttribute("baseUrl") == null) ? "./browse" : request.getAttribute("baseUrl").toString() %>?path=<%=encode(fileInfo.getAbsolutePath(), "UTF-8")%>"><%= escapeHtml(fileInfo.getName()) %></a>
-                  </th>
-                  <th><%= fileInfo.getSize() %></th>
-                  <th><%= fileInfo.getBlockSizeBytes() %></th>
-                  <th>
-                    <% if (fileInfo.getIsDirectory()) { %>
-                    <% } %>
-                    <% if (!fileInfo.getIsDirectory()) { %>
-                      <% if (fileInfo.getInMemory()) { %>
-                        <i class="icon-hdd"></i>
-                      <% } %>
-                      <% if (!fileInfo.getInMemory()) { %>
-                        <i class="icon-hdd icon-white"></i>
-                      <% } %>
-                      <%= fileInfo.getInMemoryPercentage() %>%
-                    <% } %>
-                  </th>
-                  <% if ((Boolean)request.getAttribute("showPermissions")) { %>
-                    <th><%= fileInfo.getMode() %></th>
-                    <th><%= fileInfo.getOwner() %></th>
-                    <th><%= fileInfo.getGroup() %></th>
-                  <% } %>
-                  <th><%= (fileInfo.getPersistenceState()) %></th>
-                  <th><%= (fileInfo.isPinned() ? "YES" : "NO") %></th>
-                  <th><%= fileInfo.getCreationTime() %></th>
-                  <th><%= fileInfo.getModificationTime() %></th>
-                  <% if ((Boolean) request.getAttribute("debug")) { %>
-                    <th><%= fileInfo.getId() %></th>
-                    <th><% for (String location : fileInfo.getFileLocations()) { %>
-                          <%= location %> <br/>
-                        <% } %>
-                    </th>
-                  <% } %>
-                </tr>
-              <% } %>
-            <% } %>
-          </tbody>
-        </table>
-
-        <%@ include file="pagination-component.jsp" %>
-
-      </div>
+        <div id="MainCtrl" ng-controller="MainCtrl">
+          <div id="grid1" ui-grid="gridOptions" ui-grid-pagination ui-grid-resize-columns ui-grid-move-columns class="grid"></div>
+            <button id='toggleFiltering' ng-click="toggleFiltering()" class="btn btn-success" style="margin-top: 5px;">Toggle Filtering</button>
+          </div>
+        </div>
     </div>
   </div>
   <%@ include file="footer.jsp" %>
 </div>
-
-<%@ include file="browse-pagination-header.jsp" %>
-<%@ include file="pagination-control.jsp" %>
```

Comment:
- `browse-pagination-header.jsp` and `memory-pagination-header.jsp` are not used and I've removed it.
- `pagination-control.jsp` are used by blockinfo.